ivc/src/ivc/columns.rs

```diff
@@ -38,6 +38,11 @@ pub type IVCPoseidonColumn = PoseidonColumn<IVC_POSEIDON_STATE_SIZE, IVC_POSEIDO
 ///
 ///```text
 ///
+///         Inputs:
```

Comment:
The following comments are cherry-picked in https://github.com/o1-labs/proof-systems/pull/2338